models/pathology_tumor_detection/docs/README.md

```diff
@@ -112,6 +113,7 @@ The model needs to be used with NVIDIA hardware and software. For hardware, the
 [1] He, Kaiming, et al, "Deep Residual Learning for Image Recognition." In Proceedings of the IEEE conference on computer vision and pattern recognition, pp. 770-778. 2016. <https://arxiv.org/pdf/1512.03385.pdf>
```

Comment:
Hi @drbeh , should the `Intended Use` section and the license of clara be removed?